### PR TITLE
code review

### PR DESCRIPTION
**Pull request by whiteWolfx99**
# Recipe Project

## Description
- This project is for the microverse and includes HTML code for a restaurant menu.

## Thanks!.

### CODE REVIEW

**Comment by whiteWolfx99** [comment]
# Hello @whiteWolfx99 👋 
### I'm Hevar, your PR reviewer
You still have some issues to work out before moving on to the next project, but you're almost there!

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966

## Status: REQUIRED CHANGES ♻️

- [ ] So far, so good. Your work is excellent. I looked into your project and found that you should include a README file and use can use this template. [Click here](https://github.com/whiteWolfx99/Last-Microverse/blob/main/README.md)
- [ ] If you put the CSS in a separate file and connect to the html file like that, it will work much better.. [Click here](https://github.com/whiteWolfx99/Last-Microverse)
- [ ] No linter is present. Don't forget to include linter in your project. [Click here](https://github.com/microverseinc/linters-config/tree/master/html-css)

## Optional suggestions

_[OPTIONAL]_ Use much more meaningful file name.

Happy coding! 👏

If something is unclear, feel free to post any queries or comments in the PR thread.
Please don't forget to cite me as @whiteWolfx99 in your query so that I get notified.

#### Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: Wings.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+        font-size: 20px;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="container" style="text-align: center">
+      <h1 id="header" style="margin-left: 40px">
+        Air Fryer Keto Chicken Wings
+      </h1>
```
Comment (whiteWolfx99) [inline Wings.html:21]: You should change the div tag to header tag.

This way will be semantically correct and the screen readers and other technologies will thank you. +1

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <title>odin-recipes</title>
+
+    <style>
+      #li {
+        padding-left: 10px;
+        text-decoration: none;
+        font-style: italic;
+
```
Comment (whiteWolfx99) [inline index.html:12]: Delete all the blank lines.

eyes Remember the "Best practices" followed in Microverse:
Be careful with blank lines and indentation.

Do not add blank lines, spaces, or indentations without a reason.
For readability, add blank lines to separate large or logical code blocks.
For readability, add two spaces of indentation. Do not use the tab key.
on some code editors, you can set the tab button to do an X amount of spaces.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          <li style="list-style-type: none">
+            <a id="li" style="color: green" href="recipes\Wings.html"
+              >Air Fryer Keto Chicken Wings</a
+            >
+          </li>
+          <li style="list-style-type: none">
+            <a id="li" style="color: green" href="recipes\Meatloaf.html"
+              >Air-Fried Meatloaf</a
+            >
+          </li>
```
Comment (whiteWolfx99) [inline index.html:47]: You should have separate file for the CSS.

File: index.html (hunk 2 above)
Comment (awais-amjed) [reply index.html:12]: For example: Here "Delete all the blank lines" sounds like an order. It'd be better to rephrase it like "Kindly delete all the blank lines"

Kindly go through all the comments and try to rephrase the sentences into suggestions 🤗

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @whiteWolfx99 , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗

## Highlights

- [x] Good Description ✔️
- [x] Nice Use of inline comments ✔️

gif

## Required Changes ♻️

- [ ] Please try to phrase the comments politely, as suggestions instead of orders.
  For example:
    - This is a polite suggestion: "I think it would be better to use semantic tags"
    - This reads as an order: "Use semantic tags"
- [ ] Check the reply to the comments above to better understand 🤗

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (christianbiring1) [reply index.html:47]: - [ ] As said by the previous reviewer, Kindly make sure the comments are polite, as suggestions instead of orders.

**Comment by whiteWolfx99** [comment]
Hi @christianbiring1
thanks for telling me whe i made mistake so i there any thing else that i have to change it ?

**Comment by whiteWolfx99** [comment]
hi @christianbiring1 
is there anything else that i should fixed ?
best regards

**Review by jasonfelice — APPROVED** [review]
Hi @whiteWolfx99, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
Professional Communication✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**